packages/web-app-files/src/components/SideBar/Details/FileDetailsMultiple.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div id="oc-file-details-sidebar">
```

Comment:
Please make the id unique. E.g. `oc-file-details-multiple-sidebar`. This helps us with e.g. testing if the correct panel is shown.